directjngine/src/test/java/com/softwarementors/extjs/djn/router/processor/standard/StandardErrorResponseDataTest.java

```diff
@@ -18,19 +18,32 @@
  *
  * You should have received a copy of the GNU Lesser General Public License
  * along with DirectJNgine.  If not, see <http://www.gnu.org/licenses/>.
- * 
- * This software uses the ExtJs library (http://extjs.com), which is 
+ *
+ * This software uses the ExtJs library (http://extjs.com), which is
  * distributed under the GPL v3 license (see http://extjs.com/license).
  */
-
-package com.softwarementors.extjs.djn.test.java;
+package com.softwarementors.extjs.djn.router.processor.standard;
 
 import org.testng.annotations.Test;
 
-public class JavaTestsWorkingTest {
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class StandardErrorResponseDataTest
+{
+  @Test
+  public void validateActionEncoded() {
+    //  public StandardErrorResponseData( Long tid, String action, String method, Throwable exception, boolean debugOn ) {
```

Comment:
Is this comment really needed?

Reply:
heh, tis not :)

Reply:
https://github.com/sonatype/directjngine/pull/10/commits/8f08a20132903acccb0347f62861a525174467a8